src/pspm_hb2hp.m

```diff
@@ -93,8 +93,9 @@
 newdata.header.chantype = 'hp';
 
 o.msg.prefix = 'Heart beat converted to heart period and';
-[sts, winfos] = pspm_write_channel(fn, newdata, options.channel_action, o);
-if nsts == -1
+try
+    [~,winfos] = pspm_write_channel(fn, newdata, options.channel_action, o);
+catch
```

Comment:
I assume previously it was a typo to check `nsts == -1`. I am in favor of using try catch here, but we still should check for the status flag of `pspm_write_channel`. Maybe put an assertion after calling `pspm_write_channel` inside the try block?

Reply:
Apart from this, I think it looks good. Feel free to merge it afterwards.

Reply:
Got it, thanks for the review !